Test.py

```
@@ -0,0 +1 @@
+Test
```

Ruff (0.15.21)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)